frontend/openchat-agent/src/services/user/mappers.ts

```diff
@@ -1014,3 +1016,28 @@ function formatIcrc1Account(candid: ApiIcrc1Account): string {
 
     return subaccount !== undefined ? `${owner}:${subaccount}` : owner;
 }
+
+export function deletedMessageResponse(candid: ApiDeletedDirectMessageResponse): DeletedDirectMessageResponse {
+    if ("Success" in candid) {
+        return { 
+            kind: "success",
+            content: messageContent(candid.Success.content, "unknown"),
```

Comment:
Sender is "unknown"? Isn't the sender on the message response? We need the userId not the username.

Reply:
It's only needed for the crypto content type which cannot be deleted. The response returns the MessageContent not the full Message.

Reply:
Changed